mail_debrand/readme/CONTRIBUTORS.rst

```diff
@@ -1 +1,7 @@
+* Rafael Blasco <rafabn@tecnativa.com>
+* Jairo Llopis <jairo.llopis@tecnativa.com>
+* Jordi Ballester Alomar <jordi.ballester@eficent.com>
+* Darshan Patel <darshan.patel.serpentcs@gmail.com>
```

Comment:
As the module was totally redone, contributors were reset, and this one hasn't contributed in current one.